### ISSUE TITLE
Python error after recent IPMI update

### ISSUE THREAD

**Issue by Gelioks** [jhpope/smc_ipmi#3]
Hi,

I've recently updated Supermicro IPMI/BMC [firmware](https://www.supermicro.com/en/products/motherboard/X12SPM-TF) and now I'm getting the following error when the script runs:
```
Traceback (most recent call last):
  File "/GitHub/Server/Configs/SMC-IPMI/scripts/smc_ipmi.py", line 106, in <module>
    pmbus_points = parse_pminfo(pminfo_out, args.temp_unit)
  File "/GitHub/Server/Configs/SMC-IPMI/scripts/smc_ipmi.py", line 74, in parse_pminfo
    tag = f'sensor=PMBus_{module}_' + row[0].strip().replace(' ', '\ ')
UnboundLocalError: local variable 'module' referenced before assignment
```
Please let me know if you need further details.

Thanks.

**Comment by jhpope** [comment]
Hi @UniverseXXX,

Can you please submit the output to the command:
`/opt/smcipmitool/SMCIPMITool '{YOUR.IP}' 'username' 'password' pminfo`

Please make sure to redact your IP, username, password, etc from anything you submit.

**Comment by Gelioks** [comment]
```
➜ /GitHub/Server/Configs/SMC-IPMI/app/SMCIPMITool XXX.XXX.XXX.XXX XXXXXXX XXXXXXXX pminfo
[Module 1]
 Item                           |                          Value
 ----                           |                          -----
 Status                         |               [STATUS OK](00h)
 Input Voltage                  |                       246.75 V
 Input Current                  |                         0.27 A
 Main Output Voltage            |                        12.12 V
 Main Output Current            |                         3.08 A
 Temperature 1                  |                        31C/88F
 Temperature 2                  |                       46C/115F
 Fan 1                          |                       3648 RPM
 Fan 2                          |                          0 RPM
 Main Output Power              |                           37 W
 Input Power                    |                           55 W
 PMBus Revision                 |                           0x22
 PWS Serial Number              |                XXXXXXXXXXXXX
 PWS Module Number              |                    PWS-802A-1R
 PWS Revision                   |                            1.0

[Module 2]
 Item                           |                          Value
 ----                           |                          -----
 Status                         |               [STATUS OK](00h)
 Input Voltage                  |                        246.5 V
 Input Current                  |                         0.27 A
 Main Output Voltage            |                        12.13 V
 Main Output Current            |                         4.07 A
 Temperature 1                  |                        33C/91F
 Temperature 2                  |                       48C/118F
 Fan 1                          |                       3424 RPM
 Fan 2                          |                          0 RPM
 Main Output Power              |                           49 W
 Input Power                    |                           55 W
 PMBus Revision                 |                           0x22
 PWS Serial Number              |                XXXXXXXXXXXX
 PWS Module Number              |                    PWS-802A-1R
 PWS Revision                   |                            1.0
```

@jhpope  Many thanks for looking into this.

**Comment by jhpope** [comment]
@UniverseXXX can you test the fix in #4

**Comment by Gelioks** [comment]
@jhpope just tested it and all seems working fine now. Many thanks for fixing this.
